javascript/javascript1/week1/Age-ify.js

```diff
@@ -0,0 +1,4 @@
+const yearOfBirth = 1985;
+let yearFuture = 2040;
+let age = yearFuture - yearOfBirth;
+console.log("you will be " + age + " years old in " + yearFuture);
```

Comment:
you could use back ticks (`${}`) like this  to combine string and variable. (go through this how to combine the variable and string otherway around https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)

Reply:
done. thank you.